sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/MemberShouldBeStatic.CSharp9.cs

```diff
@@ -0,0 +1,23 @@
+﻿
+int TopLevelMethod() => 0;              // FN
+
+var localVariable = 42;
+int LocalFunction() => localVariable;   // Compliant
```

Comment:
I think we can add one more case with static lambdas to document the behavior.

Reply:
There's `New RSPEC: Use static lambdas when possible.` note in the spreadsheet for lambdas. And since lambda is not a member, I didn't added as it's not relevant for this rule.

Reply:
In order to document the behavior we can still add a case to show that this rule will not trigger for those cases and that it's a decision we took not a case we missed.